models/ed/R/met2model.ED2.R

```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```

Comment:
Yeah! I think I just pushed it. Should I submit another pull request?

On Thu, May 21, 2015 at 3:39 PM, Michael Dietze <notifications@github.com>
wrote:

> In models/ed/R/met2model.ED2.R
> <https://github.com/PecanProject/pecan/pull/490#discussion_r30838963>:
>
> > @@ -267,7 +267,7 @@ for(year in start_year:end_year) {
> >
> >    ## write DRIVER file
> >    sites <- 1
> > -  metgrid <- c(1,1,1,1,lon,lat)
> > +  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
>
> Ann, if you could change back that one line and push it to Github I'd be
> happy to pull the rest. Thank's for the update!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PecanProject/pecan/pull/490/files#r30838963>.
>